src/index.js

```diff
@@ -1,26 +1,101 @@
-import list from './list.js';
+import Lists from './modules/list.js';
 import './style.css';
 import more from './more.png';
 import recycle from './recycle.png';
 import left from './to-left.png';
+import del from './delete.jpeg';
 
-function component() {
-  list.sort((a, b) => a.index - b.index);
+const List = new Lists();
 
+const renderList = (description, id, completed) => {
+  const listDiv = document.createElement('div');
+  listDiv.classList.add('eachList');
+  listDiv.innerHTML = `
+    <div class="left">
+      <input type="checkbox" id="check-${id}" ${completed ? 'checked' : ''}>
+      <span class="des ${completed ? 'completed' : ''}" contenteditable="true" data-id="${id}">${description}</span>
+    </div>
+    <img class="more" src="${more}" alt="more" data-id="${id}">
+    <img class="removebtn" src="${del}" alt="remove" style="display:none;" data-id="${id}">
+  `;
+  const newDiv = document.createElement('div');
+  newDiv.classList.add('eachList');
+  newDiv.dataset.id = id;
+  newDiv.innerHTML = listDiv.innerHTML;
+
+  const checkbox = newDiv.querySelector(`#check-${id}`);
+  checkbox.addEventListener('change', () => {
+    if (checkbox.checked) {
+      newDiv.querySelector('.des').classList.add('completed');
+      List.completelist(id + 1);
+    } else {
+      newDiv.querySelector('.des').classList.remove('completed');
+      List.uncompletelist(id + 1);
+    }
```

Comment:
-[Optional] (Mandatory) @beteland123 I noticed that whenever I check a task as completed it strikes through and it's key `completed ` in the local storage turns true as it should be, How ever whenever I reload the page, the strike-through disappears, but local storage retains its completed as true, please make sure that on reload every completed task still retains their strike through the class.  

##### Video showing the scenario (on reload)

[Screencast from 05-13-2023 10:58:27 PM.webm](https://github.com/beteland123/Todo-list/assets/58771507/97f84383-31aa-4d7b-8b6a-c0c0033a720d)